PCHR-2032: Move leave requests style to LA

igorpavlov opened the pull request:
## Overview

This PR removes SCSS styling from the theme. The styling is moved to https://github.com/civicrm/civihr/pull/1978.

## Technical Details

1. Removed the styling file
2. Added a rule to import styling from L&A.

## Comments

No tests needed.

---

- [ ] Tests Pass

deb1990 commented:
@igorpavlov  After pulling latest for la3 in shoreditch, please run the gulp job in this theme. There should be some updates. If yes please push those too. Because in my previous PR I fixed some styling with ui-select in shoreditch. But I cannot see those fixes in SSP.

igorpavlov commented:
@deb1990 

> After pulling latest for la3 in shoreditch, please run the gulp job in this theme. There should be some updates. If yes please push those too. Because in my previous PR I fixed some styling with ui-select in shoreditch. But I cannot see those fixes in SSP.

I have just:
1) pulled la-milestone-3 - mine local repo was up to date
2) rebased 2032 with la-milestone-3 - was up to date

What should I do?

igorpavlov-zz commented:
@deb1990 

> @igorpavlov After pulling latest for la3 in shoreditch, please run the gulp job in this theme. There should be some updates. If yes please push those too. Because in my previous PR I fixed some styling with ui-select in shoreditch. But I cannot see those fixes in SSP.

Done.
